src/packages/excalidraw/README.md

```diff
@@ -286,7 +286,7 @@ This helps to load Excalidraw with `initialData`. It must be an object or a [pro
 | --- | --- | --- |
 | `elements` | [ExcalidrawElement[]](https://github.com/excalidraw/excalidraw/blob/master/src/element/types.ts#L78) | The elements with which Excalidraw should be mounted. |
 | `appState` | [AppState](https://github.com/excalidraw/excalidraw/blob/master/src/types.ts#L37) | The App state with which Excalidraw should be mounted. |
-| `scrollToCenter` | boolean | This attribute implies whether to scroll to the center once Excalidraw is mounted. By default, it will not scroll to the center. |
+| `scrollToCenter` | boolean | This attribute implies whether to scroll to the center once Excalidraw is mounted. By default, it will not scroll to the center. Make sure you pass `initialData.appState.scrollX` and `initialData.appState.scrollY` so it scrolls to center correctly |
```

Comment:
Actually, this makes me realize the API is named wrong. It doesn't scroll to scene center, but instead centers to nearest elements. That's why we need `appState.scrollX/Y`.

So we should either rename to `scrollToContent`, or make the function really scroll to center (whatever that means in our case), in which case we should disregard the `appState.scrollX/Y`.

We should make `appState.scrollX/Y` optional in even the former case.

Reply:
@dwelle as per the current API lets keep it `scrollToCenter` and we can update it in next version. For now i can update the description that it will scroll nearrest element to center. Same we should also change the ref method its also `setScrollToCenter`